tests/integration/buildcmd/test_build_terraform_applications.py

```diff
@@ -432,3 +439,40 @@ def test_build_and_invoke_lambda_functions(self, function_identifier):
                 "multiValueHeaders": None,
             },
         )
+
+
+@skipIf(
+    not (CI_OVERRIDE and not IS_WINDOWS),
+    "Skip Terraform test cases unless running in CI and not in Windows",
+)
+class TestUnsupportedCases(BuildTerraformApplicationIntegBase):
+    terraform_application = Path("terraform/unsupported")
+
+    @parameterized.expand(
+        [
+            (
+                "conditional_layers",
+                r"SAM CLI could not process a Terraform project that contains Lambda functions that are linked to more than one lambda layer",
```

Comment:
we should change this error message to say "AWS SAM CLI"

Reply:
sure, will update the error message.